### PR TITLE
bugfix:pk too long in lock table on db mode and optimize error log.

### LINKED ISSUE #1679: Seata client error Global lock wait timout
- [ ] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description

seata-all 0.8.1 版本AT db模式下主键长度过长（大于32位），导致 Global lock wait timout

### Ⅱ. Describe what happened

 db模式下 使用36位的UUID设置主键时，报错Global lock wait timout，

```
Just paste your stack trace here!
```

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1. xxx
2. xxx
3. xxx

### Ⅵ. Environment:

- JDK version : 
- OS :
- Others:

### PR DESCRIPTION
**Pull request by jsbxyyx**
### Ⅰ. Describe what this PR did
pk too long in lock table on db mode and optimize error log.

### Ⅱ. Does this pull request fix one issue?

fixes #1679

### CODE REVIEW

File: server/src/main/java/io/seata/server/lock/db/DataBaseLocker.java [hunk 1]
```diff
@@ -86,6 +91,8 @@ public boolean releaseLock(List<RowLock> locks) {
     public boolean isLockable(List<RowLock> locks) {
         try {
             return lockStore.isLockable(convertToLockDO(locks));
+        } catch (StoreException e) {
```
Comment (slievrly) [inline server/src/main/java/io/seata/server/lock/db/DataBaseLocker.java]: can throw StoreException？

File: server/src/main/java/io/seata/server/AbstractTCInboundHandler.java [hunk 2]
```diff
@@ -217,7 +253,12 @@ public GlobalStatusResponse handle(GlobalStatusRequest request, final RpcContext
             @Override
             public void execute(GlobalStatusRequest request, GlobalStatusResponse response)
                 throws TransactionException {
-                doGlobalStatus(request, response, rpcContext);
+                try {
+                    doGlobalStatus(request, response, rpcContext);
+                } catch (StoreException e) {
+                    throw new TransactionException(TransactionExceptionCode.FailedStore,
+                            String.format("global status store failed. xid=%s, msg=%s", request.getXid(), e.getMessage()));
```
Comment (slievrly) [inline server/src/main/java/io/seata/server/AbstractTCInboundHandler.java]: store failed ->  request failed

File: core/src/main/java/io/seata/core/store/db/LogStoreDataBaseDAO.java [hunk 3]
```diff
@@ -369,7 +370,7 @@ public boolean deleteGlobalTransactionDO(GlobalTransactionDO globalTransactionDO
             }
             return rets;
         } catch (SQLException e) {
-            throw new StoreException(e);
+            throw new DataAccessException(e);
```
Comment (l81893521) [inline core/src/main/java/io/seata/core/store/db/LogStoreDataBaseDAO.java]: is it should be StoreException?

Comment (l81893521) [reply core/src/main/java/io/seata/core/store/db/LogStoreDataBaseDAO.java]: my fault.

File: core/src/main/java/io/seata/core/exception/TransactionExceptionCode.java [hunk 4]
```diff
@@ -114,6 +114,11 @@
...
      */
     FailedWriteSession,
+
+    /**
+     * Failed to store exception code
+     */
+    FailedStore
     ;
 
     private static final Map<Integer, TransactionExceptionCode> MAP = new HashMap<>(values().length);
```
Comment (xingfudeshi) [inline core/src/main/java/io/seata/core/exception/TransactionExceptionCode.java]: values().length*2 to avoid resize.

Comment (xingfudeshi) [inline core/src/main/java/io/seata/core/exception/TransactionExceptionCode.java]: values().length*2 to avoid resize of HashMap.

Comment (jsbxyyx) [reply core/src/main/java/io/seata/core/exception/TransactionExceptionCode.java]: This time there is no change here, is it necessary to modify it in this submission?

Comment (xingfudeshi) [reply core/src/main/java/io/seata/core/exception/TransactionExceptionCode.java]: > This time there is no change here, is it necessary to modify it in this submission?

Please amend it by the way.

Comment (jsbxyyx) [reply core/src/main/java/io/seata/core/exception/TransactionExceptionCode.java]: ok

**Pull request merged**